lib/inv_gmresdr_quda.cpp

```diff
@@ -21,219 +21,249 @@
 #include <Eigen/Dense>
 #include <Eigen/Eigenvalues>
 
-
 /*
-GMRES-DR algorithm:
-R. B. Morgan, "GMRES with deflated restarting", SIAM J. Sci. Comput. 24 (2002) p. 20-37
-See also: A.Frommer et al, "Deflation and Flexible SAP-Preconditioning of GMRES in Lattice QCD simulations" ArXiv hep-lat/1204.5463
+  GMRES-DR algorithm:
```

Comment:
A lot of reformatting of this file.  Any reasoning behind this?  Since @alexstrel  is working on cleaning up his deflation algorithms, this will likely lead to a lot of conflicts for him.

Reply:
I'm pretty sure that's just a clang format.

Reply:
Sure, but clang-format should only be passed to the diff from develop.  This looks to be touching lots of untouched lines.

Reply:
Looks like I haven't 'clanged' this yet. I'll apply the formatting script, should bring it in line with develop.

Reply:
Hmm. I guess not re clang. I did have to touch this file to rename eigensolver variables. Not sure why so many lines got changed.